### PR TITLE
enable gunicorn behind nginx reverse proxy + ssl

### PR DESCRIPTION
**Pull request by YggNode**

### CODE REVIEW

File: resync.py [hunk 1]
```diff
@@ -68,12 +68,21 @@ def changeDownloadUrl(rssFeed, serverURL, domainName, logger):
...
+
     confFile = open(os.getcwd() + "/annexes.yml", 'r')
     serverConfiguration = yaml.safe_load(confFile)
     confFile.close()
 
-    if not (os.path.exists(os.getcwd() + "/logs/")):
-        os.mkdir(os.getcwd() + '/logs')
+
 
     logging.basicConfig(level=logging.INFO, filename=os.getcwd() + "/resync.log")
```
Comment (LimeDrive) [inline resync.py:87]: Timestamps on the logs, will be betters to checks the logs file. 
Check also to remove all print() from the code, no more shell access whit gunicorn
```py
logging.basicConfig(format='%(levelname)s - %(asctime)s ::   %(message)s', datefmt='%d/%m/%Y %I:%M:%S', level=logging.INFO, filename=os.getcwd() + "/resync.log")
```

File: yggnode.py [hunk 2]
```diff
@@ -117,13 +117,6 @@ def getStatus():
 
     return renderTxt
 
+# need to keep ?
```
Comment (LimeDrive) [inline yggnode.py:120]: Yes for debug:
debug=True and localhost

File: init.sh [hunk 3]
```diff
@@ -23,35 +21,38 @@ sudo docker run -d \
   -e LOG_LEVEL=info \
   --restart unless-stopped \
```
Comment (LimeDrive) [inline init.sh:22]: --restart on-failure

Comment (YggNode) [reply init.sh:22]: Based on https://github.com/YggNode/FlareSolverr, it seems better to keep initial restart tag.

Comment (YggNode) [reply init.sh:22]: anyway, it doesn't change fundamental behavior, so i add your modification.

**Pull request closed without merging**